packages/react/src/context/slash-id-context.tsx

```diff
@@ -16,6 +16,7 @@ export type StorageOption = "memory" | "localStorage";
 
 export interface SlashIDProviderProps {
   oid: string;
+  onOidChange?: (oid: string) => void | Promise<void>
```

Comment:
Are we sure we need this? Maybe a direct callback could be registered on the hook itself?

Reply:
It's not really needed to be honest, but equally neither is the reactive behaviour for the `oid` prop using `useEffect`.

I'm happy to treat `oid` as `initialOid` and be done with it, we can rename later.